Makefile

```
@@ -19,7 +19,7 @@ GOLANGCI_LINT=github.com/golangci/golangci-lint/v2/cmd/golangci-lint@v2.1.6
 BUF_CLI_VERSION=v1.54.0
 
 ifndef PACKAGE_VERSION
-	PACKAGE_VERSION=0.1.0-SNAPSHOT
+	PACKAGE_VERSION=0.1.0+SNAPSHOT
```

Please no.  In [semver terms](https://semver.org/#spec-item-10), `0.1.0` comes _before_ `0.1.0+SNAPSHOT`.  Unfortunately Python got this [hideously wrong](https://packaging.python.org/en/latest/specifications/version-specifiers/#version-specifiers).

I suggest

```suggestion
	PACKAGE_VERSION=0.0.0    # (Python pre-releases are incompatible with semver)
```

It is still wrong and broken.  But at least it is clearly smaller than any version number we have ever released.  And it's legal semver _and_ Python version.